bundles/org.openhab.binding.irobot/README.md

```diff
@@ -12,18 +12,21 @@ without any need for a dedicated MQTT server.
 
 ## Discovery
 
-Roombas on the same network will be discovered automatically, however in order to connect to them a password is needed. The
-password is a machine-generated string, which is unfortunately not exposed by the original iRobot smartphone application,
-but it can be downloaded from the robot itself. If no password is configured, the Thing enters "CONFIGURATION PENDING" state.
-Now you need to perform authorization by pressing and holding the HOME button on your robot until it plays series of tones
-(approximately 2 seconds). The Wi-Fi indicator on the robot will flash for 30 seconds, the binding should automatically
-receive the password and go ONLINE.
-
-After you've done this procedure you can write the password somewhere in case if you need to reconfigure your binding. It's
-not known, however, whether the password is eternal or can change during factory reset.
+Roombas on the same network will be discovered automatically, however in order to connect to them a password is needed.
+The password is a machine-generated string, which is unfortunately not exposed by the original iRobot smartphone application, but it can be downloaded from the robot itself.
+If no password is configured, the Thing enters "CONFIGURATION PENDING" state.
+Now you need to perform authorization by pressing and holding the HOME/DOCK button on your robot until it plays series of tones (approximately 2 seconds).
+The Wi-Fi indicator on the robot will flash for 30 seconds, the binding should automatically receive the password and go ONLINE.
+
+After you've done this procedure you can write the password somewhere in case if you need to reconfigure your binding.
+It's not known, however, whether the password is eternal or can change during factory reset.
 If you have issues getting the password make sure there are no other devices like your smartphone communicating with the robot.
 You can also try using [these python scripts](https://github.com/NickWaterton/Roomba980-Python) to get the password.
 
+*note* For file based configuration storing the password is essential.
+Once the password is populated in the Thing properties, you must copy that into the [config files](#irobotthings-example) in order for it to persist.
```

Comment:
It won't be in Thing properties, but it can be found in the code tab, e.g.:
```yaml
UID: irobot:roomba:roomba960
label: Roomba
thingTypeUID: irobot:roomba
configuration:
  ipaddress: 192.168.1.63
  password: xxx
```

Reply:
Thanks, adjusted. Please check if this is clear enough, i struggle a little with the sentence.